modules/serverless-tag-to-digest-resolution.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies
+//
+// * serverless/admin_guide/serverless-configuration.adoc
+
+:_content-type: CONCEPT
+[id="serverless-tag-to-digest-resolution_{context}"]
+= Tag-to-digest resolution
+
+If the Knative Serving controller has access to the container registry, Knative Serving resolves image tags to a digest when you create a revision of a service. This is referred to as _tag-to-digest resolution_, and helps to provide consistency for deployments.
+
+If your container registry uses a self-signed certificate, you must configure the Knative Serving controller so that it trusts that certificate. To do this, you must create a config map or secret and then configure controller custom certificates by modifying the `controller-custom-certs` spec in the `KnativeServing` custom resource (CR). The config map or secret must reside in the same namespace as the Knative Serving CR. If no controller custom certificate is specified, this setting defaults to use the `config-service-ca` config map.
```

Comment:
Yes that was my thinking too. Makes sense we need to list available options.

Reply:
We shouldn't add references like  _See "Configuring a custom PKI" in the "Additional Resources" section for more details_, because this is in a module and then we have to check constantly to ensure that it's accurate to what's in the assembly. Other than that I'm OK with these updates.

Reply:
P.S. another reason why is that the PKI docs don't exist yet in the OSD doc set, so we'd be referring to a link that isn't there in those docs (they use the same source modules).

Reply:
Thanks. Looking good now.